uk.co.compucorp.civicrm.tasksassignments/js/test/tasks-assignments/services/document.service.spec.js

```diff
@@ -42,7 +42,7 @@ define([
         Document.save({
           action: 'create'
         }, {json: documentMock.single() || {}}
-      );
+        );
```

Comment:
there shouldn't be a space here.

Reply:
I think it should be present.  Otherwise the indentation was wrong before. To avoid confusion changed it to
```javascript
Document.save({
  action: 'create'
}, {json: documentMock.single() || {}});
```